skglm/experimental/quantile_regression.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from numba import float64
+from skglm.datafits import BaseDatafit
+from skglm.utils.prox_funcs import ST_vec
+
+
+class Pinball(BaseDatafit):
+    r"""Pinball datafit.
+
+    The datafit reads::
+
+        sum_i quantile * max(y_i - Xw_i, 0) + (1 - quantile) * max(Xw_i - y_i, 0)
```

Comment:
The naming "quantile" is unfortunate. It should be "quantile level", i.e. the quantile at level 50% is the median. The "quantile" in the formula is not the quantile, but Xw_i is an estimation of the quantile.

Reply:
thanks @lorentzenchr for the insightful remark, we'll change that